.github/workflows/nightly-jdk8.yml

```diff
@@ -0,0 +1,28 @@
+name: Nightly cross-validate and test
+
+on:
+  schedule:
+    - cron:  '0 4,21 * * *'
+
+jobs:
+  compile-and-test:
+    name: Compile and test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@$1.8.0
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.2
+
+      - name: Compile and test for JDK ${{ matrix.JABBA_JDK }}, Scala ${{ matrix.SCALA_VERSION }}
+        run: |-
+          cp .jvmopts-ghactions-nightly-jdk8 .jvmopts
+          sbt clean update whitesourceUpdate whitesourceCheckPolicies test:compile test publishLocal publishM2
```

Comment:
I see in a lot of places: `publishLocal publishM2`, but I think it should be performed a maximum of 1 time

Reply:
I'm duplicating what we had in Jenkins at the moment. 

Plus, publishing, compared to testing, takes no time.